plugin/src/index.ts

```diff
@@ -8,8 +8,13 @@ const Portals = registerPlugin<PortalsPlugin>('Portals', {
   ios: () => import('./ios').then(m => new m.PortalsIOS())
 });
 
+/**
+ * Provides access to any initial state provided by the native application.
+ * If the web application is running in a Portal, this will always be defined
+ * with the name property.
+ * */
 export function getInitialContext<T = unknown>(): InitialContext<T> | undefined {
-  return (window as any).portalInitialContext as { name: string, value: T; };
+  return (window as any).portalInitialContext;
```

Comment:
The explicit casting here is not needed and nothing happens at runtime if this isn't a valid cast anyway.